api/hasura/actions/_handlers/createEpoch.ts

```diff
@@ -213,4 +230,42 @@ async function checkOverlappingEpoch({
   }
 }
 
+export function findSameDayNextMonth(
+  start: DateTime,
+  { week }: { week: number }
```

Comment:
curious about these two diff args types, one is object (w/ one field) one is straight straight param

Reply:
@CryptoGraffe this is just Typescript notion. I'm destructuring the param I need on the left side and defining the param on the right.

Reply:
What I meant is why doesn't the function just consistently take the arg it needs (eg start and week) so the destructuring or whatever would happen in the caller. Not important just curious why.